### ISSUE TITLE
Issue to run the game between TextGame and Home class.

### ISSUE THREAD

**Issue by davng115** [SWEInstructor/CSE1321-Project2#10]
_ On the Home class, it is just a constructor, and there is no return type. I think it won't be able to print anything in this class from the TextGame. 

_ In the TextGame, I didn’t see any object for Home Class so I’m not quite sure how can we get all the code from Home class to TextGame class.

_ When I input "home" it loops back to the beginning where the statement
"NAME + has decided to take a trip back to their home, probably to give up their useless and incredibly stupid adventure.
Do you want to control the life of a college student user again? Enter Y for yes and N for no."

_ When I input "Y" or "N" of the bellow statement :
"Do you want to control the life of a college student user again? Enter Y for yes and N for no."
_ Both options loops back to the beginning, instead "Y" should go back, and "N" should end the program and print "GAME OVER", but both of them return to the statement :
"We begin out story by taking a look at a specific college student, down on his luck, doing some brain storming on what to do about the money problem of college life
What would this college students name be?: "

_ To sum up, there is no way I can move to the next step of the game, I did try other command. Bur after that, it will always loops back to the beginning.

@ncampb39 @anhhdng @rshamsi05 @cbell108 @Rande147
